tests/test_split.py

```diff
@@ -260,27 +260,6 @@ def do_query(
         },
         False,
     ),  # Valid query but not enough columns to split.
-    (
```

Comment:
This condition now cannot happen since we are inlining aliases. The result is that the column split query will always be valid even if we remove expressions that were defining aliases.

Reply:
Original case:
the query was something like:
```
SELECT ... stuff .... transform (an ugly expression) AS projects
ORDER BY projects
```

The column splitter would build this query out of that 
```
SELECT project, event_id, timestamp, 
ORDER BY projects
```
`Projects` now is not declared and the query fails. (old and new system)

So I introduced a check in the column splitter that ensured the transformed query was still semantically valid (at least in terms of columns requested and aliases).
Now that we are expanding the aliases I was not able anymore to make the column splitter generate a broken query, since the aliased expression is preserved.

Reply:
This makes sense, thanks for clarifying.